src/components/Main.jsx

```diff
@@ -58,6 +59,31 @@ export default function Main(WrappedComponent, title) {
   }, [])
   const dispatch = useDispatch()
   const alertData = useSelector((state) => state.alert)
+  
+  //Dark theme context
+  const {theme} = useContext(ThemeContext)
+
+  
+  const body = document.querySelector('body')
+  theme === 'dark' ? body.style.transition = 'all 1s ease' : body.style.transition = 'all 1s ease'
```

Comment:
why are both sides of this if statement doing the same thing?

Reply:
I don't remember exactly why I did this since I wrote this a while back, but I'm guessing I was thinking I wanted to have a smooth transition from dark to light and the other way around. Good catch, the conditional statement doesn't do anything lol. I must've been tired or something; I'll fix it!